app/src/main/java/es/situm/gettingstarted/fetchresources/FetchResourcesActivity.kt

```diff
@@ -0,0 +1,145 @@
+package es.situm.gettingstarted.fetchresources
+
+import android.os.Bundle
+import android.view.View
+import android.widget.AdapterView
+import android.widget.ArrayAdapter
+import com.google.android.material.snackbar.Snackbar
+import es.situm.gettingstarted.R
+import es.situm.gettingstarted.common.SampleActivity
+import es.situm.sdk.SitumSdk
+import es.situm.sdk.communication.CommunicationConfig
+import es.situm.sdk.communication.CommunicationConfigImpl
+import es.situm.sdk.configuration.network.NetworkOptions
+import es.situm.sdk.configuration.network.NetworkOptionsImpl
+import es.situm.sdk.error.Error
+import es.situm.sdk.model.cartography.*
+import es.situm.sdk.utils.Handler
+import kotlinx.android.synthetic.main.activity_fetch_resources.*
+
+class FetchResourcesActivity : SampleActivity() {
+
+    private val cacheStrategies = NetworkOptions.CacheStrategy.values().also { it.sortBy { item -> item.toString() } }
+    var selectedCacheStrategy: NetworkOptions.CacheStrategy? = NetworkOptions.CacheStrategy.TIMED_CACHE
```

Comment:
No parece que esta variable vaya a ser nunca nula
```suggestion
    var selectedCacheStrategy: NetworkOptions.CacheStrategy = NetworkOptions.CacheStrategy.TIMED_CACHE
```